index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Document</title>
+</head>
+<body>
+  <div id="background-mobile">
+      <div id="status-bar"></div>
+      <header>
+        <h2>Coder</h2>
```

Comment:
- [ ] Kindly put your logo inside an `a` tag to make it click and that will improve the user's experience. you got this 💯